client/components/app-promo/index.jsx

```diff
@@ -69,15 +75,26 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		const element = (
 			<div className="app-promo">
 				<span tabIndex="0" className="app-promo__dismiss" onClick={ this.dismiss } >
 					<Gridicon icon="cross" size={ 24 } />
 					<span className="screen-reader-text">{ this.translate( 'Dismiss' ) }</span>
```

Comment:
Another `className should follow CSS namespace guidelines (expected app-promo__ prefix)`.
A quick search shows that nowhere in the entire project a `.screen-reader-text` element has another class to conform to that guideline.

Note: `className="app-promo__something screen-reader-text"` gets rid of the warning.